.github/CODEOWNERS

```diff
@@ -532,6 +532,7 @@
 # User’s profile page.
 /core/controllers/profile*.py @vojtechjelinek
 /core/templates/domain/user/ @vojtechjelinek
+/core/templates/domain/profile/ @vojtechjelinek
```

Comment:
Where is this used?

Reply:
There is a ProfileSumamryObjectFactory. It is used in both creator and learners dashboard

Reply:
Should we move this to the `domain/user/` dir? (COnsidering it's related to users and `user` sounds more like a domain name.)

Reply:
done